docs/_components/search.md

```diff
@@ -9,15 +9,16 @@ lead: >
   Search allows users to search for specific content if they know what search terms to use or can’t find desired content in the main navigation.
 ---
 
+{% include helpers/base-component.html component="search" %}
+
 ## Default
+
 Search should be easily found. For the default search version, consider displaying in the header and/or side navigation.
 
 {% capture example %}
-<form accept-charset="UTF-8" action="https://search.usa.gov/search" class="usa-search usa-search--smalldisplay-flex flex-justify-center tablet:grid-col-4" method="get" role="search">
-  <input name="utf8" type="hidden" value="&#x2713;"/>
-  <input name="affiliate" type="hidden" value="login.gov"/>
-  <label class="usa-sr-only" for="search-field-header-nav">Search</label>
-  <input class="usa-input" id="search-field-header-nav" name="query" type="search">
+<form class="usa-search" role="search">
+  <label class="usa-sr-only" for="tkBH">Search</label>
```

Comment:
i'm not used to these randomized IDs, is that a new thing we're doing?

Reply:
> i'm not used to these randomized IDs, is that a new thing we're doing?

It predates me, I've just kept it going 🤷  We would want unique IDs at least for each demo example, though I'm not sure the best way to include that in code snippets. USWDS goes the route of creating unique-but-meaningful IDs ([example](https://designsystem.digital.gov/components/text-input/)), which I think would work a bit better, though require more (up-front) effort. I've found these randomized IDs to be a bit of a pain actually, since I don't know that there's an easy way to enforce it, or even how best to create new ones.

Reply:
yeah the red code in the diff had the meaningful ones so that's why I was wondering why the random ones came back

Reply:
> yeah the red code in the diff had the meaningful ones so that's why I was wondering why the random ones came back

Ah, I see now. I might still be in favor of the revisions, since:

- The IDs were copied verbatim from the brochure site. "heading" doesn't really apply in the context of this example.
- Consistency for consistency's sake